doc/hooks/waf.section.md

```diff
@@ -1,3 +1,36 @@
 # wafHook {#wafhook}
 
-Overrides the configure, build, and install phases. This will run the “waf” script used by many projects. If `wafPath` (default `./waf`) doesn’t exist, it will copy the version of waf available in Nixpkgs. `wafFlags` can be used to pass flags to the waf script.
+[Waf](https://waf.io) is a Python-based software building system.
+
+In Nixpkgs, `wafHook` overrides the default configure, build, and install phases.
+
+## Variables controlling wafHook
+
+### `wafPath` {#wafPath}
+
+Location of the `waf` tool. It defaults to `./waf`, to honor software projects that include it directly inside their source trees.
+
+If `wafPath` doesn't exist, then `wafHook` will copy the `waf` provided from Nixpkgs to it.
```

Comment:
```suggestion
If `wafPath` doesn't exist and `./waf` doesn't exist, then `wafHook` will copy the `waf` provided from Nixpkgs to it.
```

Reply:
According to the hook script (and reflected by this documentation), wafPath is set to `./waf` when not set by the user. So, that suggestion would be redundant.

Further, the hook script does not try to find a "hardcoded" path, it only verifies if `wafPath` exists.

It is useful when, by any reason, the upstream-provided waf is not desired: just set `wafPath = ./nixpkgs-waf` and be happy.